ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: news_post
+title: "CVE-2024-27280: StringIO에서 버퍼 초과 읽기 취약점"
+author: "hsbt"
+translator: "shia"
+date: 2024-03-21 4:00:00 +0000
+tags: security
+lang: ko
+---
+
+버퍼 초과 읽기 취약점에 대한 보안 수정이 포함된 StringIO gem 버전 3.0.1.1과 3.0.1.2를 릴리스했습니다.
+이 취약점에는 CVE 식별자 [CVE-2024-27280](https://www.cve.org/CVERecord?id=CVE-2024-27280)이 할당되었습니다.
+
+## 세부 내용
+
+Ruby 3.0.x의 3.0.6까지와 3.1.x의 3.1.4까지 함께 배포된 StringIO 3.0.1에서 문제가 발견되었습니다.
```

Comment:
How about this?

```suggestion
Ruby 3.0.x부터 3.0.6까지, 3.1.x부터 3.1.4까지 함께 배포된 StringIO 3.0.1에서 문제가 발견되었습니다.
```

Reply:
👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)